IntelliTect.TestTools.TestFramework/IntelliTect.TestTools.TestFramework.Tests/TestBuilderTests/PositiveConditions/ResolverTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using IntelliTect.TestTools.TestFramework.Tests.TestData.Dependencies;
+using System;
+using Xunit;
+
+namespace IntelliTect.TestTools.TestFramework.Tests.TestBuilderTests.PositiveConditions
+{
+    public class ResolverTests
+    {
+        [Fact]
+        public void Test1()
```

Comment:
These tests could use better names

Reply:
D'oh. Yeah, definitely meant to change those and then just forgot... Fixed!